tests/kernel/mellanox_ofed.pm

```diff
@@ -36,7 +36,7 @@ sub run {
         zypper_call('--quiet in openvswitch python-devel python-libxml2 libopenssl1_0_0 insserv-compat libstdc++-devel createrepo rpm-build kernel-syms tk', timeout => 500);
     }
     else {
-        die "OS VERSION not supported. Available only on 15 and 12-SP4";
+        die "OS VERSION not supported. Available only on 15, 15-SP1 and 12-SP4";
```

Comment:
`15 SP{0-1}` is better and will allow you to save some space when we will have SP2/3/4 :D

Reply:
good point